docs/concepts/Dialog-Command.md

```diff
@@ -0,0 +1,105 @@
+# DIALOG COMMAND
+
+`askx dialog` opens an interactive mode, similar to Node's Read-Eval-Print-Loop (REPL), in which developers can simulate a multi-turn conversation with Alexa. For each utterance, Dialog command leverages [simulate-skill](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-command-reference.html#simulate-command) to asynchronously poll simulation result until it's available and shows Alexa text responses.
+
+**PRE-REQUISITES:**
+
+* Dialog command only supports Custom skill type.
+* Ensure the locale you want to simulate exists in your publishingInformation in skill manifest.
+* Skill must have been enabled for simulation before running the dialog command.
+
+
+**STRUCTURE OF DIALOG COMMAND:**
+
+`askx dialog [-s|--skill-id <skill-id>] [-l|--locale <locale>] [-g|--stage <stage>] [-r|--replay <file-path>] [-p|--profile <profile>] [--debug]`
+
+**OPTIONS DESCRIPTION:**
+
+**skill-id**: Optional. Represents the Id associated with the skill the user is expecting to simulate. It should be in the format `amzn1.ask.skill.12345678-1234-1234-123456789123`. If skill-id is not specified, and if the `--replay` option is not specified, this command must be run from the skill project root directory. Do not specify the skill-id if you use the --replay option.
+
+**locale**: Optional if ASK_DEFAULT_DEVICE_LOCALE environment variable is set, otherwise, required if the `--replay` option is not used.
+
+**stage**: Optional. Indicates stage of the skill. Use development or live as values. Defaults to development.
+
+**replay**: Optional. Specify a replay file to simulate a conversation with your skill. Specifying this option will run the command in Replay Mode.
+
+**profile**: Optional. Specify a profile name to be used. Defaults to ASK_DEFAULT_PROFILE.
```

Comment:
-> `Defaults to use "default" as the profile name, if this option or environmental variable "ASK_DEFAULT_PROFILE" is not set.`